### DISCUSSION TITLE
Ignore non-SemVer tags

### DISCUSSION THREAD

**Discussion by till** [goreleaser/goreleaser#3708] [Q&A] [answered]
Hi,

I'm trying to add goreleaser to another project, but it contains tags that are not semver. So goreleaser errors. Is there any way to ignore them?

```
❯ git tag -l
dev-v0.90.64
dev-v0.90.76
dev-v0.90.78
dev-v0.90.78-alpha
dev-v0.90.79
dev-v0.90.80
untagged-4b1cd42048450a168b46
v0.1
v0.12
v0.13
v0.14
v0.15
v0.151
v0.161
v0.17
v0.18
v0.2
v0.21
v0.22
v0.23
v0.24
v0.241
v0.242
v0.243
v0.25
v0.26
v0.27
v0.27.1
v0.27.2
v0.27.3
v0.27.4
v0.27.7
v0.27.8
v0.27.9
v0.28
v0.28.1
v0.28.2
v0.28.3
v0.28.4
v0.28.5
v0.28.6
v0.28.7
v0.28.9
v0.28.91
v0.29
v0.29.1
v0.29.2
v0.29.3
v0.29.5
v0.29.6
v0.29.7
v0.29.8
v0.29.9
v0.30
v0.32
v0.33
v0.34
v0.35
v0.36
v0.37
v0.38
v0.4
v0.41
v0.42
v0.43
v0.44
v0.45
v0.46
v0.47
v0.48
v0.49
v0.5
v0.51
v0.52
v0.54
v0.55
v0.56
v0.57
v0.58
v0.59
v0.6
v0.61
v0.64
v0.66
v0.67
v0.68
v0.69
```

**Comment by caarlos0** [comment, marked as answer]
nope, you'll need semver.

you can probably create a new semver tag on the latest tag you have, e.g. `v0.69.0` pointing to the same commit as `v0.69`, and that should work if your next tags are also semver.

**Reply by till** [reply]
Guess that means I have to delete all the junk in there. 😅

**Reply by caarlos0** [reply]
not really, just recreate the last tag... goreleaser won't look back on all tags, just the current and previous ones

**Reply by till** [reply]
@caarlos0 maybe that is the problem though. It failed at the `dev-` tags for some reason.

I deleted all those:

```
dev-v0.90.64
dev-v0.90.76
dev-v0.90.78
dev-v0.90.78-alpha
dev-v0.90.79
dev-v0.90.80
untagged-4b1cd42048450a168b46
```

And it works.
